patch/kernel/rockchip64-current/rockpi-4b_increase_rx_delay.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/arch/arm64/boot/dts/rockchip/rk3399-nanopi4.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-nanopi4.dtsi
```

Comment:
Could you please make the patch name consistent with the ones that are already there.
`board-rockpi4-dts-rx-delay-tweak.patch` would fit better what we already have IMHO ;-)